.changeset/changelog-lit.cjs

```diff
@@ -0,0 +1,116 @@
+const {
+  getInfo,
+  getInfoFromPullRequest,
+} = require('@changesets/get-github-info');
+
+// Forked from: https://github.com/atlassian/changesets/blob/main/packages/changelog-github/src/index.ts
```

Comment:
nbd but would be nice in future to add a commit before this one which is the pristine version, so that we can see the diff

(I did a diff manually and lgtm)

Reply:
yeah, good idea